explainer.md

```diff
@@ -150,6 +151,20 @@ For entryTypes with a fixed amount of entries, like Paint Timing, using the poll
 At the same time, the callback is more useful for cases where there is varying amount of entries of the given entryType, like in Resource Timing.
 The web performance monitoring service can process all the performance information while the page is still running and only has to do very minimal work when the data needs to be reported.
 
+## Page lifetime issues
+When this API was originally designed, documents had a relatively simple lifecycle: they were
+loaded when the user navigated to them, and unloaded when the user navigated away, with the
+JavaScript environment being torn down at that time. Since then, the sitution has become more
+complex, with many browsers introducing a back-forwards-cache, with which a user can return
+a document wich they have previously navigated away from. The web has also seen a rise in
+popularity of Single Page Apps, where what appears to the user to be a navigation is actually
+just a change in state of a running page. In both of these situations, a navigation (or what
+appears to the user as a navigation) can occur without the performance timeline being reset.
```

Comment:
Should we refer to a "same-document navigation" here?

Reply:
I'm not sure if that would be confusing - not all same-document navigations are "user-perceived navigations" (fragment navs, for instance), and not all user-perceived navigations are same-document navigations (bfcache restores are not)

Was there wording that you had in mind?

Reply:
nah, let's land as is, and clarify later if confusion arises.